xml/src/test/resources/configs/docs/getting-started.xml

```diff
@@ -22,6 +22,7 @@
   <!-- tag::gettingStarted[] -->
   <cache alias="foo"> <!--1-->
     <key-type>java.lang.String</key-type> <!--2-->
+    <value-type>java.lang.String</value-type> <!--2-->
```

Comment:
It is. Since it's a offheap store, you need to specify the type. The callout is on purpose because it's the same comment for both